tests/unit/debug/deprecate-test.js

```diff
@@ -182,4 +182,38 @@ ID     foo
 UNTIL  5.0.0
 URL    https://example.com`);
   });
+
+
+  it('throws an deprecation if the current Ember-CLI version is greater than the until version of deprecation', function () {
+    expect(() => {
+      deprecate('The `foo` method is deprecated.', false, {
```

Comment:
The period is in the message here but also in the concatenated message in the implementation of `deprecate`

Reply:
Thanks, Yes removed it